src/libraries/System.Runtime.Numerics/src/System/Number.BigInteger.cs

```diff
@@ -1109,6 +1210,26 @@ public static int OmittedLength(int index)
                 return (MaxPartialDigits * (1 << index)) >> 5;
             }
 
+            public static void FloorBufferSize(int size, out int bufferSize, out int maxIndex)
```

Comment:
Is it required to calculate exact buffer length? Can it be relaxed, and let the algorithm to strip unnecessary zeros?

Reply:
Is there a concise way to calculate the inexact buffer length? It would be most concise to find out from the predefined buffer length.